apps/jenkins/upgrade/run_litmus_test.yml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: litmus-jenkins-upgrade-
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: upgrade-jenkins
+   
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+
+        env: 
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays, actionable, default
+            value: default
+
+            # Application label
+          - name: APP_LABEL
+            value: 'app=jenkins-app'
+
+            # Application namespace
+          - name: APP_NAMESPACE
+            value: jenkins
+
+            # Either deployment or statefulset
+          - name: DEPLOY_TYPE
+            value: deployment
+
+            # Specified image name for application
+          - name: IMAGE_NAME
+            value:   
```

Comment:
@vibhor995 Can we specify default image for app upgrade?

Reply:
@nsathyaseelan we can only specify the default image name when we are sure that our existing app is going to deployed with certain image .